app/controllers/chargeback_report_controller.rb

```diff
@@ -5,71 +5,46 @@ class ChargebackReportController < ApplicationController
   after_action :set_session_data
 
   include Mixins::SavedReportPaging
+  include Mixins::GenericListMixin
   include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
-    @table_name ||= "chargeback_report"
+    @table_name ||= "miq_report_result"
   end
 
-  def x_show
-    @explorer = true
+  def self.model
+    MiqReportResult
   end
 
-  def tree_select
-    self.x_active_tree = params[:tree] if params[:tree]
-    self.x_node = params[:id]
-    get_node_info(x_node)
-    replace_right_cell
+  def show_list
+    options = {:named_scope => :with_saved_chargeback_reports, :no_checkboxes => true}
+    process_show_list(options)
+    @title = _("Chargeback Saved Reports")
   end
 
-  def explorer
-    @breadcrumbs = []
-    @explorer    = true
-    build_accordions_and_trees
-
-    @right_cell_text = _("All Saved Chargeback Reports")
-    render :layout => "application" unless request.xml_http_request?
-  end
-
-  def title
-    @title = _("Chargeback Reports")
+  def show
+    @center_toolbar = 'chargeback_report'
```

Comment:
What about using the `toolbar` method?

Reply:
Cannot use toolbar method here, because toolbar is only needed on summary screen, if i use toolbar method that tries to render toolbar for list screen as well and blows up.

Reply:
I think that should work with a single argument, maybe it needs a little bit of extra research. Should I look into this more?

Reply:
sure if you have time, I tried that but that did not work for me. i don't think there is an easier way to do that.